packages/api/src/interfaces/cli/PaymentsFlat.cli.ts

```diff
@@ -0,0 +1,14 @@
+import { StaticImplements } from "../../decorators/staticImplements.decorator";
+import { CliStaticInterface } from "../../@types";
+
+import paymentsFlatService from "../../modules/data-viz/paymentFlat/paymentFlat.service";
+
+@StaticImplements<CliStaticInterface>()
+export default class PaymentsFlatCli {
+    static cmdName = "payments-flat";
+
+    async resync() {
+        console.log("Resyncing payments flat collection");
```

Comment:
Dans cli controler, on a un logger qu'il vaut mieux utiliser que console.log, car ça permet de créer des fichiers de logs, tu le trouveras dans this.logger

Reply:
merci pour l'info! Modifié!